tools/pipelines/templates/include-conditionally-run-stress-tests.yml

```diff
@@ -80,4 +80,4 @@ stages:
             testCommand: ${{ package.testCommand }}
             env:
               FUZZ_STRESS_RUN: true
-              FLUID_TEST_REPORT: 1
+              FLUID_TEST_MULTIREPORT: 1
```

Comment:
Unnecessary now?

Reply:
yeah. I need to merge in the other change and resolve the conflict.  This line will be remove.